test/library/packages/OrderedSet/general/types/testNilableBorrowed.chpl

```diff
@@ -5,7 +5,7 @@ use OsetTest;
 
 type T = testClass;
 
-proc <(lhs: borrowed T?, rhs: borrowed T?) {
+operator <(lhs: borrowed T?, rhs: borrowed T?) {
```

Comment:
Following the convention in this PR, I'd expect this one to be methods on `T` or `testClass`

It got me wondering, though, can we define secondary operator methods on type aliases? Can we do that for types returned from functions?

```chapel
record R {}
proc getType() type {
  return R; 
}

operator getType().+(lhs: R, rhs: R) { }
// or
operator getType().-(lhs: getType(), rhs: getType()) { }
// or another combination where we mix types, type aliases and type functions
```

Reply:
Seems like it works!